### PR TITLE
Bump to v0.15.0

### PR DESCRIPTION
**Pull request by hjin2902**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.
Bump to v0.15.0

## Modification
Please briefly describe what modification is made in this PR.
change_log.md and getting_started.md

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.
No

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ The required versions of MMCV, MMDetection and MMSegmentation for different vers
 
 | MMDetection3D version | MMDetection version | MMSegmentation version |    MMCV version     |
 |:-------------------:|:-------------------:|:-------------------:|:-------------------:|
-| master              | mmdet>=2.12.0          | mmseg>=0.14.1 | mmcv-full>=1.3.2, <=1.4|
-| 0.14.0              | mmdet>=2.10.0, <=2.11.0| mmseg>=0.13.0 | mmcv-full>=1.3.1, <=1.4|
+| master              | mmdet>=2.14.0| mmseg==0.14.1 | mmcv-full>=1.3.1, <=1.4|
```
Comment (Tai-Wang) [inline docs/getting_started.md:15]: Why is mmcv-full>=1.3.8 [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/__init__.py#L20)?

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
 ## Changelog
 
+### v0.15.0 (1/7/2021)
+
+#### Highlights
+
+- Support [PAConv](https://arxiv.org/abs/2103.14635)
+- Support monocular/multi-view 3D detector [ImVoxelNet] (https://arxiv.org/abs/2106.01178) on KITTI
```
Comment (Tai-Wang) [inline docs/changelog.md:8]: \[ImVoxelNet\]\(https://arxiv.org/abs/2106.01178)

File: docs/changelog.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
...
+- Support Transformer-based 3D detection method [Group-Free-3D](https://arxiv.org/abs/2104.00678) on ScanNet
+- Add documentation for tasks including LiDAR-based 3D detection and point-based 3D semantic segmentation
```
Comment (Tai-Wang) [inline docs/changelog.md:10]: Also vision-based 3D detection after merging #669

File: docs/changelog.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+- Add dataset documents like ScanNet
+- Refine docs for tutorials, Quick Run and Useful Tools
```
Comment (Tai-Wang) [inline docs/changelog.md:12]: Can remove this item (refinement itself does not need highlight)

File: docs/changelog.md [hunk 5]
```diff
@@ -1,5 +1,45 @@
...
+
+#### New Features
+
+- Support Group-Free-3D on ScanNet (#539)
+- Support PAConv modules(#598, #599)
```
Comment (Tai-Wang) [inline docs/changelog.md:17]: Add a blank space before '('

File: docs/changelog.md [hunk 6]
```diff
@@ -1,5 +1,45 @@
...
+- Support ImVoxelNet on KITTI (#627, #654)
+
+#### Improvements
+
+- Add unit tests for pipeline functions `LoadImageFromFileMono3D`, `ObjectNameFilter` and `ObjectRangeFilter` (#615)
+- Enhance [IndoorPatchPointSample](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/pipelines/transforms_3d.py) (#617)
+- Refactoring model initialization methods based MMCV (#622)
```
Comment (Tai-Wang) [inline docs/changelog.md:24]: Refactor (to be consistent with other verbs)

File: docs/changelog.md [hunk 7]
```diff
@@ -1,5 +1,45 @@
...
+- Add Chinese docs (#629)
+- Add documentation for LiDAR-based 3D detection (#642)
+- Unifying intrinsic and extrinsic matrices for all datasets (#653)
```
Comment (Tai-Wang) [inline docs/changelog.md:27]: Unify

File: docs/changelog.md [hunk 8]
```diff
@@ -1,5 +1,45 @@
...
+- Add documentation for point-based 3D semantic segmentation (#663)
+- Add documentation of ScanNet for 3D detection (#664)
+- Refine docs for tutorials (#666)
```
Comment (Tai-Wang) [inline docs/changelog.md:30]: Add documentation for vision-based 3D detection (#669)

File: docs/changelog.md [hunk 9]
```diff
@@ -1,5 +1,45 @@
...
+- Refine docs for Quick Run and Useful Tools (#686)
+
+
+#### Bug Fixes
+
+- Fix the bugs of [BackgroundPointsFilter](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/pipelines/transforms_3d.py) (#609)
```
Comment (Tai-Wang) [inline docs/changelog.md:36]: to be more specific, summarize the bug

File: docs/changelog.md [hunk 10]
```diff
@@ -1,5 +1,45 @@
...
+- Fix LoadMultiViewImageFromFiles to be compatible with DefaultFormatBundle (#611)
+- Fix the potential bug in [analyze_logs](https://github.com/open-mmlab/mmdetection3d/blob/master/tools/analysis_tools/analyze_logs.py) (#634)
```
Comment (Tai-Wang) [inline docs/changelog.md:38]: to be more specific

File: docs/changelog.md [hunk 11]
```diff
@@ -1,5 +1,45 @@
...
+- Fix test command in docs and made some refinements (#635)
```
Comment (Tai-Wang) [inline docs/changelog.md:39]: test commands ...... make some ......

**Review by Tai-Wang — CHANGES_REQUESTED** [review]
Please also modify the versions indicated in the [mminstall requirements](https://github.com/open-mmlab/mmdetection3d/blob/master/requirements/mminstall.txt).

File: docs/changelog.md [hunk 12]
```diff
@@ -1,5 +1,45 @@
...
+- Add documentation of ScanNet for 3D detection (#664)
+- Refine docs for tutorials (#666)
+- Add documentation for vision-only 3D detection (#669)
+- Refine docs for Quick Run and Useful Tools (#686)
+
+
+#### Bug Fixes
+
+- Fix the bug of [BackgroundPointsFilter](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/pipelines/transforms_3d.py) using the bottom center of ground truth (#609)
+- Fix LoadMultiViewImageFromFiles to get list format for DefaultFormatBundle (#611)
```
Comment (Tai-Wang) [inline docs/changelog.md:37]: add a link to LoadMultiViewImageFromFiles

File: docs/changelog.md [hunk 13]
```diff
@@ -1,5 +1,45 @@
 ## Changelog
 
+### v0.15.0 (1/7/2021)
+
+#### Highlights
+
```
Comment (ZwwWayne) [inline docs/changelog.md:6]: Need to add a section for compatibility to tell users why we need mmcv 1.3.8 following here https://github.com/open-mmlab/mmdetection/pull/5488/files

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**